src/Widgets/GridBreakpointsValuesWizard.php

```diff
@@ -96,19 +96,19 @@ public function generate()
     {
         $this->strGridPreset = $this->activeRecord->grid_preset;
         $this->arrGridBreakpoints = [
-            ['name' => 'all', 'label' => 'Général', 'required'=>true],
+            ['name' => 'all', 'label' => 'Général', 'required' => true, 'value' => 2],
             ['name' => 'xxs', 'start' => 0, 'stop' => 619, 'label' => 'XXS'],
             ['name' => 'xs', 'start' => 620, 'stop' => 767, 'label' => 'XS'],
             ['name' => 'sm', 'start' => 768, 'stop' => 991, 'label' => 'SM'],
             ['name' => 'md', 'start' => 992, 'stop' => 1199, 'label' => 'MD'],
             ['name' => 'lg', 'start' => 1200, 'stop' => 1399, 'label' => 'LG'],
             ['name' => 'xl', 'start' => 1400, 'stop' => 0, 'label' => 'XL'],
-        ]; /** @todo - make it generic per grid */
+        ]; /* @todo - make it generic per grid */
```

Comment:
Remove this comment, useless since there won't be more grid presets

Reply:
Done